### PR TITLE
Run streaming tests in Node TFLite tasks too

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

File: taskcluster/tc-asserts.sh [hunk 1]
```diff
@@ -42,10 +42,14 @@ assert_correct_inference()
   else
       echo "!! Non matching output !!"
       echo "got: <${phrase}>"
-      echo "xxd:"; echo "${phrase}" | xxd
+      if [ -x "$(command -v xxd)" ]; then
```
Comment (lissyx) [inline taskcluster/tc-asserts.sh:45]: we should just install `xxd` package

Comment (reuben) [reply taskcluster/tc-asserts.sh:45]: It's not available in the repo for this image.

Comment (reuben) [reply taskcluster/tc-asserts.sh:45]: (For Ubuntu 16.04 at least)

Comment (lissyx) [reply taskcluster/tc-asserts.sh:45]: it's in buster: https://packages.debian.org/search?keywords=xxd

**Pull request merged**